modules/core/src/main/resources/scalafix-migrations.conf

```
@@ -104,5 +104,12 @@ migrations = [
     newVersion: "1.0.0",
     rewriteRules: ["https://raw.githubusercontent.com/typelevel/cats/v2.1.1/scalafix/rules/src/main/scala/fix/Cats_v1_0_0.scala"],
     doc: "https://github.com/typelevel/cats/blob/v2.1.1/scalafix/README.md"
+  },
+  {
+    groupId: "org.typelevel",
+    artifactIds: ["cats-parse"],
+    newVersion: "0.3.0",
+    rewriteRules: ["martijnhoekstra/catsparsescalafix/Parser01Fix"],
```

It seems the `github:` scheme is missing here.
```suggestion
    rewriteRules: ["github:martijnhoekstra/catsparsescalafix/Parser01Fix"],
```